src/components/dashboard.css

```diff
@@ -0,0 +1,20 @@
+#finance-table > div:first-of-type,
```

Comment:
it doesn't sound right to have an ID called `finance-table` in a file called dashboard.css. Each component should have it's own scss file.